packages/gatsby/cache-dir/gatsby-browser-entry.js

```diff
@@ -57,6 +57,19 @@ const useStaticQuery = query => {
     )
   }
   const context = React.useContext(StaticQueryContext)
+
+  // query is a stringified number like `3303882` when wrapped with graphql, If a user forgets
+  // to wrap the query in a grqphql, then casting it to a Number results in `NaN` allowing us to
+  // catch the misuse of the API and give proper direction
+  if (isNaN(Number(query))) {
```

Comment:
Would it be ok to only show this error in development? Error messages are "expensive" in production code. @blainekasten

Reply:
Hmm good question. Would this ever really leak to production?

Reply:
Is it just a request to wrap it in a `process.env.NODE_ENV !== 'production'` ?